### ISSUE TITLE
Wrong OpenPGP key used when sending signed-only email

### ISSUE THREAD

**Issue by tigernero79** [thunderbird/thunderbird-android#4914]
I have recently noticed that if I set the sending of an email with the openpgp signature only option, I don't have the possibility to choose which identity to use.  k9 v5.718 uses the first signature key available in openkeychain although it is not even that of the email from which I send.  then an email is sent with the address xxx @ xxx signed by a yyyu @ yyy identity how to choose which signature key to use?  I have no choices

**Comment by GNUDimarik** [comment]
@tigernero79 how did you check which key was used?

**Comment by GNUDimarik** [comment]
nevermind. I configured openpgp for thundebird on windows and I can see now keys are the same on the both addrs

**Comment by tigernero79** [comment]
> @tigernero79 how did you check which key was used?

for informational purposes, I noticed doing application tests.  I have my private Key generated on yubico 5 Nfc, to which I then associated 4 other identities (email) obviously on yubico token there is only one email but on the Key servers updated with the other identities thanks to gnupgp and Cleopatra.  when I returned a message other than the main identity (nero79@msn.com), using another identity see (nero79@email.it) the recipient received the signature of the main identity and not the real one which was sent giving me an error signal.  these using your beautiful k9 mail application.
